src/en/authors-charm-hooks.md

```diff
@@ -129,8 +129,8 @@ services. Units of a single client service will surely want to connect to, and
 use, the same database; but if units of another client service were to use that
 same database, the consequences could be catastrophic for all concerned.
 
-If juju respected the `limit` field in relation [metadata](./authors-charm-
-metadata.html), it would be possible to work around this, but it's not a high-
+If juju respected the `limit` field in relation [metadata](./authors-charm-metadata.html),
```

Comment:
Line breaks within URLs breaks the links. Contributors to this code base need to ensure their editors are not wrapping paragraphs in a way that introduce line breaks.